assets/src/block-editor/helpers/test/addAMPAttributes.js

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
Added these unrelated tests to make up for a reduction in test coverage percentage, which was causing a ❌